package.json

```diff
@@ -50,6 +50,7 @@
     "load-grunt-tasks": "^4.0.0",
     "mocha": "^5.2.0",
     "mockery": "^2.1.0",
+    "node-abi": "^2.4.5",
```

Comment:
Instead of adding node-abi, can we just change electron-rebuild to
```
"electron-rebuild": "1.8.2"
```

Reply:
seems electron-rebuild also link to [node-abi 2.0.0 version](https://github.com/electron/electron-rebuild/blob/master/package.json#L38)